frontend/src/Settings/MediaManagement/MediaManagement.js

```diff
@@ -265,6 +272,35 @@ class MediaManagement extends Component {
                     />
                   </FormGroup>
 
+                  <FormGroup>
+                    <FormLabel>Write Metadata Tags</FormLabel>
+
+                    <FormInputGroup
+                      type={inputTypes.SELECT}
+                      name="writeAudioTags"
+                      helpText="Write metadata to the music file tags"
+                      helpTextWarning="Selecting 'Always' will alter existing files when they are imported."
+                      values={writeAudioTagOptions}
+                      onChange={onInputChange}
+                      {...settings.writeAudioTags}
+                    />
+                  </FormGroup>
+
+                  {
+                    settings.writeAudioTags.value !== 'no' &&
+                      <FormGroup>
+                        <FormLabel>Scrub Metadata Tags</FormLabel>
+
+                        <FormInputGroup
+                          type={inputTypes.CHECK}
+                          name="scrubAudioTags"
+                          helpText="Remove existing tags from files, leaving only those added by Lidarr."
+                          onChange={onInputChange}
+                          {...settings.scrubAudioTags}
+                        />
+                      </FormGroup>
+                  }
```

Comment:
Does this make more sense in Metadata section? Thats where all metadata file options are (images, nfos, etc..,)